macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionNavBarPopoverManager.swift

```diff
@@ -106,6 +107,7 @@ final class NetworkProtectionNavBarPopoverManager: NetPPopoverManager {
 
     private func statusViewSubmenu() -> [StatusBarMenu.MenuItem] {
         let appLauncher = AppLauncher(appBundleURL: Bundle.main.bundleURL)
+        let vpnAppState = VPNAppState(defaults: .netP)
```

Comment:
This existed in the closure below, but now it's used outside of the closure so I moved it up here.